src/components/Views/MapView/Map/Map.js

```diff
@@ -217,30 +217,6 @@ class Map extends React.Component<Props & StateProps & DispatchProps> {
                 : undefined
             }
           >
-            {!!observations && !isEmpty(observations)
```

Comment:
We can put this back in the code, or find some other way to address how it is currently slowing down the app. Thoughts welcome